guides/common/modules/proc_executing-a-remote-job.adoc

```diff
@@ -3,6 +3,12 @@
 
 You can execute a job that is based on a job template against one or more hosts.
 
+[NOTE]
+====
+Ansible jobs run on batches of 100 hosts, so you cannot cancel a job running on a specific host.
```

Comment:
```suggestion
Ansible jobs run in batches on multiple hosts, so you cannot cancel a job running on a specific host.
```
No reason for specific numbers :)

Reply:
I'm also wondering if the user can cancel the whole batch in mid flight.

Reply:
The best I'm willing to do about this is to create an issue for your question :) Would that work @Lennonka? (I'm just moving the note from the module I propose to delete so I'm not familiar with the background; I can only assume it's the result of a past doc bug fix.)

Reply:
Probably not necessary until a customer asks for it :) Thanks